### DISCUSSION TITLE
Diagramas de estados del manejo de categorías, manejo de sesión y manejo de usuario

### DISCUSSION THREAD

**Discussion by diegojiimenez** [TheMoys/24-25-IdSw1-SDR#32] [Q&A] [answered]
@JLRG806 @TheMoys @SantiGT1503 , Revisando los diagramas de estados y los casos de uso que se encontraron me di cuenta que probablemente faltan los diagramas de estado de los manejos de categorías, sesión y usuarios, Ya que no existen y creo que se necesitan para trabajar con el diagrama de contexto.

**Comment by TheMoys** [comment, marked as answer]
**REVISIÓN DE MANUEL 11/11/2024**

- **CASOS DE USO**
  - Unificar los casos de uso por funcionalidad y utilizar herencia para los diferentes actores que interactuan con los paquetes.
  - Para el paquete de envío de documentos, eliminar los casos de uso de "Adjuntar documento" y "Elegir tipo de confirmación" ya que esos son pasos del envío y dan pie a un diagrama de flujo.
  - El caso de uso "crear mensaje" cambiarlo por "crear envío"

- **Diagramas de estados**
   - Para el remitente, manejarlo únicamente como pendiente o al día.
   - Quitar el diagrama de alertas ya que no hay una entidad de alertas y no es necesario crearla.
   - Crear el diagrama de estado del rastreador del documento que tendrá pendiente, leido, atrasado, firmado, etc.
   - Añadir los diferentes estados del admin/responsable para el CRUD de categorías y usuarios.
